apps/yggdrasil/lib/yggdrasil/user.ex

```diff
@@ -9,6 +10,7 @@ defmodule Yggdrasil.User do
     field :hash, :string
     field :password, :string, virtual: true # not part of table
     field :password_confirmation, :string, virtual: true # not part of table
+    belongs_to :role, Yggdrasil.Role
```

Comment:
This reads weird, why `belongs_to` over `has_one`? Also we don't want a user to have many roles? I think most systems I have worked in a user could have many roles.